section-09-differential-testing/model-package/differential_tests/test_differential.py

```diff
@@ -0,0 +1,36 @@
+import math
+from pathlib import Path
+import os
+
+
+ACCEPTABLE_MODEL_DIFFERENCE = float(os.environ.get('ACCEPTABLE_MODEL_DIFFERENCE'))
+
+
+def test_difference_between_current_and_candidate_models(sample_input_data):
+    # Given
+    persisted_results_dir = Path(__file__).resolve().parent.parent
+    with open(persisted_results_dir / "current_predictions.txt", "r") as handler:
+        current_predictions = [line.rstrip() for line in handler]
+
+    with open(persisted_results_dir / "candidate_predictions.txt", "r") as handler:
```

Comment:
what is `candidate_predictions?` where do they come from? We need to think how to explain this.

Also, where current_predictions comes from as well?